src/libcore/intrinsics.rs

```diff
@@ -700,17 +700,9 @@ extern "rust-intrinsic" {
     /// which is unsafe unless `T` is `Copy`. Also, even if T is
     /// `Copy`, an all-zero value may not correspond to any legitimate
     /// state for the type in question.
+    // FIXME: deprecate once clippy's tests are fixed
```

Comment:
The "no tools shall break" freeze is basically over... I think you can just deprecate this, and we can fix clippy asynchronously.

Reply:
Will do once #62246 is merged

Reply:
Why wait? Breaking clippy *tests* brielfy is not very uncommon.

The reasong it didn't work before is that we have a one-week "freeze window" before the beta branches, where tools may not regress.

Reply:
Ah, gotcha. Done!